cypress/e2e/patient_spec/patient_location.cy.ts

```diff
@@ -0,0 +1,111 @@
+import { PatientLocation } from "@/pageObject/Patients/PatientLocation";
+import { LocationData } from "@/pageObject/Patients/PatientLocation";
+import { FacilityCreation } from "@/pageObject/facility/FacilityCreation";
+import { viewPort } from "@/utils/viewPort";
+
+const facilityCreation = new FacilityCreation();
+const patientLocation = new PatientLocation();
+
+describe("Add a new Location and associate to an Encounter", () => {
+  beforeEach(() => {
+    cy.viewport(viewPort.desktop1080p.width, viewPort.desktop1080p.height);
+    cy.loginByApi("raj");
```

Comment:
```suggestion
    cy.loginByApi("raj");
```

create a new user id , like dev nurse other than names, currently we are connected local backend of develop, so update the fixture file with new id and use them

Reply:
done